system/Database/BaseBuilder.php

```diff
@@ -3300,14 +3300,14 @@ protected function isLiteral(string $str): bool
 			return true;
 		}
 
-		static $_str;
+		static $str;
 
-		if (empty($_str))
+		if (empty($str))
 		{
-			$_str = ($this->db->escapeChar !== '"') ? ['"', "'"] : ["'"];
+			$str = ($this->db->escapeChar !== '"') ? ['"', "'"] : ["'"];
 		}
 
-		return in_array($str[0], $_str, true);
+		return in_array($str[0], $str, true);
```

Comment:
Great, it seems to be a problem here for example.

Reply:
fixed.

Reply:
Great, thanks!